### ISSUE TITLE
about_*.ps1xml pages do not render XML at all

### ISSUE THREAD

**Issue by rjmholt** [PowerShell/PowerShell-Docs#1076]
Issue Details
-----------------
Further to [#950](https://github.com/PowerShell/PowerShell-Docs/issues/950), because the raw text in `about_*` files is treated as markdown, any code goes in unescaped. With most of the PowerShell code this just leads to formatting issues. But with XML, markdown renderers try to interpret it as HTML and hide it as a result.

An example from [about_Types.ps1xml](https://msdn.microsoft.com/en-us/powershell/reference/3.0/microsoft.powershell.core/about/about_types.ps1xml):

From the raw markdown:
```text
<AliasProperty>: Defines a new name for an existing property.

The <AliasProperty> tag must have a pair of <Name> tags that specify
the name of the new property and a pair of <ReferencedMemberName> tags
that specify the existing property.

For example, the Count alias property is an alias for the Length
property of array objects.

<Type>
<Name>System.Array</Name>
<Members>
<AliasProperty>
<Name>Count</Name>
<ReferencedMemberName>Length</ReferencedMemberName>
</AliasProperty>
</Members>
</Type>
```

The HTML this converts to:
```html
<p /><aliasproperty>: Defines a new name for an existing property.<p /><p />The <aliasproperty> tag must have a pair of <name> tags that specify
the name of the new property and a pair of <referencedmembername> tags
that specify the existing property.<p /><p>For example, the Count alias property is an alias for the Length
property of array objects.</p><type><name>System.Array</name><members><aliasproperty><name>Count</name><referencedmembername>Length</referencedmembername></aliasproperty></members></type><p />
```

GitHub's rendering of the markdown:
![github_abouttypes](https://cloud.githubusercontent.com/assets/7009879/23334384/f1dfae9a-fb51-11e6-8c5d-838d228e6d18.PNG)

MSDN's rendering:
![docs_abouttypes](https://cloud.githubusercontent.com/assets/7009879/23334386/00884dee-fb52-11e6-8fa3-c8dcd26f1168.PNG)

Version(s) of document impacted
------------------------------
- [x] Impacts [5.1 document](https://msdn.microsoft.com/en-us/powershell/reference/3.0/microsoft.powershell.core/about/about_types.ps1xml)
- [x] Impacts [5.0 document](https://msdn.microsoft.com/en-us/powershell/reference/5.0/microsoft.powershell.core/about/about_types.ps1xml)
- [x] Impacts [4.0 document](https://msdn.microsoft.com/en-us/powershell/reference/4.0/microsoft.powershell.core/about/about_types.ps1xml)
- [x] Impacts [3.0 document](https://msdn.microsoft.com/en-us/powershell/reference/3.0/microsoft.powershell.core/about/about_types.ps1xml)

**Comment by rjmholt** [comment]
Fantastic -- I'll take a look

**Comment by zjalexander** [comment]
😨 

well, thanks for doing this initial analysis. Let me look into the PlatyPS pipeline to see what the heck is going on. We shouldn't have to fix all of these issues by hand so this might be easier to fix on the playtps side of things.

**Comment by zjalexander** [comment]
yeah this is all on our end and i can't blame platyps. we need an all-up scrub of about_ topics as I said in #950

**Comment by zjalexander** [comment]
@sdwheeler is working on our about_ problem and this may or may not be part of our PS6 plans, we might re-evaluate in the leadup to PS6 release.

**Comment by sdwheeler** [comment]
Fixed in PR #1914 - waiting to merge
